src/util/notifications_util.js

```diff
@@ -115,9 +116,10 @@ class Notificator {
     async _notify(fs_context, log_file, failure_append) {
         const file = new LogFile(fs_context, log_file);
         let send_promises = [];
+        let notif;
         await file.collect_and_process(async str => {
             try {
-                const notif = JSON.parse(str);
```

Comment:
why did you change this? where do you write to this value?

Reply:
The `notif` declaration is outside the `try` block so that is could be used for to get the name for the event in the `catch` block.